src/tests/ftest/get_remote_files.sh

```diff
@@ -130,7 +130,7 @@ list_tag_files() {
 #######################################
 compress_files() {
     # shellcheck disable=SC2086
-    find ${1} -maxdepth 0 -type f -size +1M -print0 | xargs -r0 lbzip2 -v
+    find ${1} -maxdepth 0 -type f -size +1M -print0 | sudo xargs -r0 lbzip2 -v
```

Comment:
does running tests already require passwordless sudo?  If not, does this add that requirement?